### PR TITLE
Feat/apply style to footer

### PR DESCRIPTION
**Pull request by Yukit02**

### CODE REVIEW

File: src/styles/main.css [hunk 1]
```diff
@@ -6,4 +6,12 @@
 
 /* body {
   background-color: rgb(30, 32, 37, 1);
-} */
\ No newline at end of file
+} */
+
+@layer utilities {
```
Comment (hoonsubin) [inline src/styles/main.css:11]: Please add the font to tailwind config file. Try not to modify the css file

File: src/footer/FooterIconList.tsx [hunk 2]
```diff
@@ -5,25 +5,25 @@ type IFooterIconListProps = {
...
 
-                .footer-icon-list :global(a:hover) {
-                    @apply text-gray-700;
+                .footer-icon-list :global(span) {
+                    @apply text-2xl xs:text-xl;
                 }
 
                 .footer-icon-list :global(svg) {
-                    @apply fill-current w-5 h-5;
+                    @apply fill-current w-10 xs:w-8 mr-2;
```
Comment (hoonsubin) [inline src/footer/FooterIconList.tsx:26]: I don't think there is a point to use xs? Break points should handle anything above md or xl

File: src/footer/CenteredFooter.tsx [hunk 3]
```diff
@@ -1,32 +1,17 @@
...
-      </ul>
-    </nav>
-
-    <div className="mt-8 flex justify-center">
+    <div className="flex justify-center">
       <FooterIconList>{props.iconList}</FooterIconList>
     </div>
 
-    <div className="mt-8 text-sm">
-      <FooterCopyright />
```
Comment (hoonsubin) [inline src/footer/CenteredFooter.tsx:27]: Please don't remove the copyright message. It's important

File: tailwind.config.js [hunk 4]
```diff
@@ -43,6 +43,14 @@ module.exports = {
                 hero: '4.5rem',
             },
         },
+        screens: {
+            xs: { max: '639px' },
+            sm: { min: '640px', max: '767px' },
+            md: { min: '768px', max: '1023px' },
+            lg: { min: '1024px', max: '1279px' },
+            xl: { min: '1280px', max: '1535px' },
+            '2xl': { min: '1536px' },
```
Comment (hoonsubin) [inline tailwind.config.js:52]: Can you add 3xl as well?

**Review by hoonsubin — CHANGES_REQUESTED** [review]
These are some changes I recommend making based on the code. I'll run this source later and add more reviews after the changes

File: tailwind.config.js (hunk 4 above)
Comment (Yukit02) [reply tailwind.config.js:52]: I can, but How minimum width?

Comment (hoonsubin) [reply tailwind.config.js:52]: `1920px` is a good start. Also, please remove the max property. Only add minimum values. You can do something like this:
```
theme: {
    screens: {
      'sm': '640px',
      // => @media (min-width: 640px) { ... }

      'md': '768px',
      // => @media (min-width: 768px) { ... }

      'lg': '1024px',
      // => @media (min-width: 1024px) { ... }

      'xl': '1280px',
      // => @media (min-width: 1280px) { ... }

      '2xl': '1536px',
      // => @media (min-width: 1536px) { ... }
    }
  }
```
I recommend that you read this article to learn about responsive designing https://responsivedesign.is/strategy/page-layout/defining-breakpoints/

File: src/styles/main.css (hunk 1 above)
Comment (hoonsubin) [reply src/styles/main.css:11]: you can add fonts like this https://tailwindcss.com/docs/font-family#customizing

**Comment by Yukit02** [comment]
I fixed it.
Please check again.

File: src/navigation/NavbarTwoColumns.tsx [hunk 5]
```diff
@@ -1,20 +1,11 @@
 import React, { ReactNode } from 'react';
 
-import Link from 'next/link';
-
 type INavbarProps = {
-  logo: ReactNode;
```
Comment (hoonsubin) [inline src/navigation/NavbarTwoColumns.tsx:6]: Why did you remove this? The Navbar prop is showing an error and the logo is not rendering

**Review by hoonsubin — CHANGES_REQUESTED** [review]

File: src/templates/Footer.tsx [hunk 6]
```diff
@@ -5,84 +5,35 @@ import Link from 'next/link';
...
-            <Link href="/">
-              <a>
-                <svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
-                  <path d="M23.998 12c0-6.628-5.372-12-11.999-12C5.372 0 0 5.372 0 12c0 5.988 4.388 10.952 10.124 11.852v-8.384H7.078v-3.469h3.046V9.356c0-3.008 1.792-4.669 4.532-4.669 1.313 0 2.686.234 2.686.234v2.953H15.83c-1.49 0-1.955.925-1.955 1.874V12h3.328l-.532 3.469h-2.796v8.384c5.736-.9 10.124-5.864 10.124-11.853z" />
-                </svg>
-              </a>
-            </Link>
-
             <Link href="/">
               <a>
```
Comment (hoonsubin) [inline src/templates/Footer.tsx:16]: Please add the link for the button

**Comment by Yukit02** [comment]
I fixed it.
Please check again.

**Review by hoonsubin — APPROVED** [review]
I guess we can make additional improvements in the next PR

**Pull request merged**